caom2pipe/tests/test_astro_composable.py

```diff
@@ -243,3 +243,31 @@ def test_check_h5():
     test_fqn_missing = '/test_files/not_there.h5'
     test_result = ac.check_h5(test_fqn_missing)
     assert not test_result, 'missing failure'
+
+
+def test_check_fits_verify():
+    files = {
```

Comment:
Add a file without `.fits` in the name here to check that method.

Reply:
Done